web/router.js

```diff
@@ -10,16 +12,54 @@ const router = new Router({
   mode: 'history',
   routes: [
     // NON-DRAWER ROUTES
+    {
+      path: '/admin',
+      meta: {
+        auth: {
+          scope: [AuthScope.ADMIN],
+        },
+        title: 'Admin Console',
+      },
+      component: () => import(/* webpackChunkName: "admin" */ '@/web/views/FcAdmin.vue'),
+      children: [{
+        path: '',
+        name: 'admin',
+        meta: {
+          auth: {
```

Comment:
I *think* we can get rid of the `meta` declarations on children here and just delegate to the parent route - let's try that out.  (if we can't, I'd say we should fix that instead of duplicating scopes on every child route!)

Reply:
yup, we can get rid of these 🎉